plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/node/provider/LocalWorkspaceFolderPathProvider.java

```diff
@@ -30,25 +30,64 @@
  */
 @Singleton
 public class LocalWorkspaceFolderPathProvider implements WorkspaceFolderPathProvider {
-    private final String projectsFolderPath;
+
+    @Inject(optional = true)
+    @Named("host.projects.root")
+    /**
+     * this value provide path to projects on local host
+     * if this value will be set all workspace will manage same projects from your host
+     */
+    private String projectsFolder;
+
+    final String workspacesFolder;
```

Comment:
It is not clear from variables names what is the purpose of them. workspacesFolder looks pretty similar to projectsFolder. Can you rename it for clarity.